plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/create-variable-product-block-editor.spec.js

```diff
@@ -232,6 +234,8 @@ test.describe( 'Variations tab', () => {
 				.getByRole( 'tab', { name: 'General' } )
 				.click();
 
+			await page.getByLabel( 'Regular price', { exact: true } ).click();
+
```

Comment:
Why is this change needed in this PR? It seems completely unrelated.

Reply:
I asked for help from Jon to fix a flaky E2E test and he asked if he could push to this branch. Agree it's unrelated, maybe I should have asked for a separate PR :)

Reply:
In the future I'd keep unrelated changes in separate PRs if possible... makes tracking down changes a bit easier. And, let's us focus more on related changes while reviewing a PR. But, this one was super small, so no worries.